client/src/stories/Buttons.stories.js

```diff
@@ -1,14 +1,44 @@
 import React from 'react';
+import styled from '@emotion/styled';
 import SubmitButton from '../components/SubmitButton';
 
 export default {
   title: 'Buttons',
 };
 
-export const SignUpSubmit = () => {
-  return <SubmitButton>Sign Up</SubmitButton>;
+const AppContainer = styled.div`
+  display: flex;
+  flex-flow: row nowrap;
+  background-color: rgba(14, 5, 46, 1);
+  width: 100vw;
+  height: 100vh;
+`;
+
+const MainContainer = styled.div`
+  display: flex;
+  flex-flow: column nowrap;
+  flex-grow: 1;
+  overflow: auto;
+  width: 100%;
+  height: 100%;
+`;
```

Comment:
don't you have these as components somewhere?

Reply:
i had the sidebar nav first inside the buttons stories. then I removed it but I can see that this add and remove is not documented now.. only the adding of the app and main containers are visible 🤔 
yeah my system with adding these containers was not very smart since I did it before adding Global Styles to storybook.. I need to fix that in another PR. 
and no I don't have appcontainer and maincontainer as components since they will be styled inside the App.js later on